hw/top_earlgrey/cdc/cdc_waivers.misc.tcl

```diff
@@ -4,76 +4,99 @@
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_intr_state_generic_tx_watermark.q[0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_txf_ctrl.rptr[1:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_upload.u_arbiter.u_req_fifo.gen_normal_fifo.u_fifo_cnt.rptr_o[2:0]") && (Signal == "top_earlgrey.u_spi_device.u_memory_2p.b_rvalid_sram_q") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_upload.u_arbiter.u_req_fifo.gen_normal_fifo.u_fifo_cnt.rptr_o[2:0]") && (Signal == "top_earlgrey.u_spi_device.u_memory_2p.b_rvalid_sram_q") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.readbuf_addr_busclk[31:0]") && (Signal == "top_earlgrey.u_spi_device.u_readcmd.readbuf_addr[31:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_rv_dm.dap.i_dmi_jtag_tap.jtag_ir_shift_q[4]") && (Signal == "IOR2") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_intr_state_generic_rx_full.q[0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[11:2]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_intr_state_generic_rx_watermark.q[0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[11:0]") && (Association == "None")} -status {Waived} -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_rv_dm.dap.i_dmi_jtag_tap.tap_state_q[3:0]") && (Signal == "IOR0") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_rxf_ptr_wptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[12:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_txf_ptr_rptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_txf_ctrl.rptr[12:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+
+
+# W_MASYNC : included in waived paths
```

Comment:
@eunchan can you take a look at the SPI ones below in this file?

Reply:
yep looks good they are fwmode waivers which the tool confuses the clock information.

Reply:
great, thanks for checking!